src/docs/concepts-development/translations/index.md

```diff
@@ -0,0 +1,243 @@
+---
+title: Translations in Flex
+slug: translations
+updated: 2022-05-13
+category: concepts-development
+ingress:
+  This article introduces translations in Flex and how they are modified
+  and edited in Flex Console
+published: true
+---
+
+In the FTW templates, user-facing content is not written directly into
+the source code. Instead, the source code uses
+[React Intl message formatting](https://formatjs.io/docs/intl#formatmessage)
+that defines keys for each meaningful piece of content, and a translator
+or a content creator can then define the message (i.e. the value) for
+each key in their language. The end user only sees the translator's
+words, not the key itself, unless the key is missing a value in the
+selected language.
+
+The key - value syntax is as follows:
+
+```json
+"<component name>.<translation key>": "<translation>"
+```
+
+For example:
+
+```json
+"ManageListingCard.editListing": "Edit listing"
+```
+
+The key is then used in the code, so that the code does not need to be
+changed even if the value ends up changing.
+
+```js
+// ManageListingCard.js uses the variable to identify the message
+<FormattedMessage id="ManageListingCard.editListing" />
+```
+
+Starting from 2022-05, marketplace operators can modify the wording of
+the translations in Flex Console. This means that operators can make
+changes to the marketplace texts without the need for code changes. In
+addition, the same translations can now be used from several different
+client applications, making it easier to make centralized changes.
+
+![Simple Console translation](./translation_simple.png)
+
+## How translations are handled in Flex
+
+With Console-editable translations, Flex introduces a concept of
+[assets](/references/) TODO: ADD LINK ONCE REFERENCE EXISTS. Assets
```

Comment:
TODO: seems to be /references/assets/
https://github.com/sharetribe/flex-docs/pull/564/files#diff-355989deba881861dc38945d9e84b18b50d92b7e32c9d96753d38649b839d3eaR3